### ISSUE TITLE
Example Dockerfile not working

### ISSUE THREAD

**Issue by jokedurnez** [rstudio/rsconnect-jupyter#175]
Compare these two Dockerfiles for jupyter or jupyterhub.  They only differ in baseimage and test-users.

```
FROM jupyterhub/jupyterhub

ARG VERSION=1.1.0.37
ARG REPOSITORY=https://s3.amazonaws.com/rstudio-rsconnect-jupyter

RUN wget ${REPOSITORY}/rsconnect-${VERSION}-py2.py3-none-any.whl
RUN pip install rsconnect-${VERSION}-py2.py3-none-any.whl && \
    jupyter-nbextension install --sys-prefix --py rsconnect && \
    jupyter-nbextension enable --sys-prefix --py rsconnect && \
    jupyter-serverextension enable --sys-prefix --py rsconnect

# create test users
RUN useradd -m -s /bin/bash user1 && \
    bash -c 'echo -en "password\npassword" | passwd user1' 

CMD ["jupyterhub"]
```

VS

```
FROM jupyter/base-notebook

ARG VERSION=1.1.0.37
ARG REPOSITORY=https://s3.amazonaws.com/rstudio-rsconnect-jupyter

RUN wget ${REPOSITORY}/rsconnect-${VERSION}-py2.py3-none-any.whl
RUN pip install rsconnect-${VERSION}-py2.py3-none-any.whl && \
    jupyter-nbextension install --sys-prefix --py rsconnect && \
    jupyter-nbextension enable --sys-prefix --py rsconnect && \
    jupyter-serverextension enable --sys-prefix --py rsconnect
```

They have the same way of installing rsconnect.  The second container (jupyter) works without problem.  With the first container (jupyterhub), I am unable to connect to a RStudio Connect Server, as in the following screenshot. 
(different permutations of the url with https, port identification,... all result in the same)

![image](https://user-images.githubusercontent.com/7630327/64897916-09a89080-d63a-11e9-9546-6be08eea33fb.png)

This is also the case in a system-install of rsconnect (without Docker).  I am looking for a minimal example of jupyterhub being able to connect to rsconnect.  Is there any update necessary in the dockerfile/jupyterhub?

**Comment by mmarchetti** [comment]
I am able to recreate this. The relevant log entry is:

```
[W 2019-09-25 17:06:33.530 JupyterHub log:158] 405 POST /rsconnect_jupyter/verify_server (@::ffff:172.17.0.1) 9.18ms
```

405 is HTTP method not allowed. This suggests that the server-side extension isn't being installed/enabled correctly.

**Comment by mmarchetti** [comment]
This was fixed in #125 which was released with v1.2.0. The current release is v1.2.1.93.
